app/code/Magento/Inventory/Test/Mftf/Test/SkippedTest.xml

```diff
@@ -8,6 +8,29 @@
 
 <tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="../../../../../../../vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Test/etc/testSchema.xsd">
+    <test name="AdminSourceSelectionAlgorithmSourcePriorityWithSimpleMsiProductTest">
```

Comment:
Please unskip this 3 tests since as I can see the bugs was fixed

Reply:
Done